pgcli/key_bindings.py

```diff
@@ -25,12 +30,15 @@ def _(event):
         Enable/Disable Multiline Mode.
         """
         _logger.debug('Detected F3 key.')
-        line.always_multiline = not line.always_multiline
+        buf = event.cli.current_buffer
+        buf.always_multiline = not buf.always_multiline
 
-    @handle(Keys.ControlSpace, in_mode=InputMode.INSERT)
+    @handle(Keys.ControlSpace, filter=~filters.HasSelection())
```

Comment:
I'm finally caught up on reviewing the new api changes in prompt_toolkit (0.26). 

The filters portion is not strictly necessary, but it doesn't hurt. What you're checking to see is whether we're in the selection mode. The only way to get into a selection mode with emacs keybindings is by hitting Ctrl+Space. But since we're overriding ControlSpace itself, I don't think we'll ever be in a selection mode. That's why I think having that arg is redundant.

/cc @jonathanslenders for a sanity check of my explanation here. No hurry, but I'd appreciate your input when you have some time. 

 But it's a non-blocker. So I'll merge this in.